File: MODUL 7/SOAL 3/PRAK603-2210817310002-MUHAMMADRIDHOPRASETYA.c
#include <stdio.h>

int main(){
    int baris1,baris2,i,isi,hasil;
    scanf("%d %d", &baris1, &baris2);
    int angka1[baris1], angka2[baris2];
    if(baris1==baris2){
        for(i=0; i<baris1; i++){
            scanf("%d", &isi);
            angka1[i]=isi;
    }
    for(i=0; i<baris2; i++){
        scanf("%d", &isi);
        angka2[i]=isi;
    }
    for(i=0; i<baris1; i++){
        hasil=angka1[i]* angka2[i];
        printf("%d ", hasil);
         }
    }
    else{
        printf("Jumlah Tidak Sama");
    }
}

File: MODUL 7/SOAL 3/PRAK603-2210817310002-MUHAMMADRIDHOPRASETYA.py
n1, n2 = map(int, input().split())
if (n1!=n2):
    print('Jumlah tidak sama')
else:
    baris1=list(map(int, input().split()))
    baris2=list(map(int, input().split()))
    for i in range (n1):
        print(baris1[i]*baris2[i], end=' ')